### ISSUE TITLE
Analyze aibolit report

### ISSUE THREAD

**Issue by olenagerasimova** [artipie/rpm-adapter#303]
Let's analyze report (check [this](https://github.com/artipie/rpm-adapter/actions?query=workflow%3A%22Experimental+CI+tools%22) action) generated by [aibolit](https://github.com/yegor256/aibolit) and try to follow its recommendations to make our code better. 
I'm proposing the following process: 
1) choose class from the report to work on, 
2) fix relevant problems and submit PR,
3) report bugs (if any) to Aibolit,
4) as for now there is no possibility to suppress bugs found by Aibolit, add comment here with small summary about the class you worked on: class name, problems left and why

Feel free to report any suggestion to Aibolit team and discuss any code related problems with me. 

Do not close this issue when first PR is closed, I will close it when whole report by Aibolit is analyzed, or we will not need it anymore.

**Comment by olenagerasimova** [comment]
@acheshkov hi, thanks for integrating Aibolit into rpm-adapter! I'd like to start analyzing Aibolit report for rpm-adapter and reporting found problems (if any) to Aibolit. Don't you mind?
@g4s8 what do you think of proposed prosecc?

**Comment by g4s8** [comment]
@olenagerasimova yes, I think you can report all problems here: https://github.com/yegor256/aibolit/
